test/mocha/cli.js

```diff
@@ -77,6 +77,24 @@ describe("bin/uglifyjs", function () {
             done();
         });
     });
+    it("should not consider source map file content as source map file name (issue #2082)", function (done) {
+        var command = [
+            uglifyjscmd,
+            "test/input/issue-2082/sample.js",
+            "--source-map", '\'content="test/input/issue-2082/sample.js.map"\'',
```

Comment:
> But this change prevents it from failing when it should!

Indeed - I forgot you want multiple `--source-map` here.

> what was the error?

The extra quotation marks are causing interoperability issues. I have tested this updated patch that will fail with original `bin/uglifyjs` & pass with this PR:
```diff
--- a/test/mocha/cli.js
+++ b/test/mocha/cli.js
@@ -81,8 +81,7 @@ describe("bin/uglifyjs", function () {
         var command = [
             uglifyjscmd,
             "test/input/issue-2082/sample.js",
-            "--source-map", '\'content="test/input/issue-2082/sample.js.map"\'',
-            "--source-map", '\'root="/"\'',
+            "--source-map", "content=test/input/issue-2082/sample.js.map",
             "--source-map", 'url=inline',
         ].join(" ");
```

(Sorry for the belated reply - I went for a nap.)